apps/opik-backend/src/main/java/com/comet/opik/domain/SpanService.java

```diff
@@ -270,11 +270,12 @@ public Mono<Long> create(@NonNull SpanBatch batch) {
                 .distinct()
                 .toList();
 
+        log.info("Creating batch of spans for projects '{}'", projectNames);
```

Comment:
Minor: there can be up to 1K project names worst case. Maybe print the size. Anyway, this is fine.